packages/web3-utils/types/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dtslint.json",
+    "rules": {
+        "semicolon": false,
+        "no-import-default-of-export-equals": false,
+        "file-name-casing": [true, "kebab-case"]
```

Comment:
file-name-casing should be pascal-case

Reply:
see my above comment around this so we can discuss 👍